Formula/circumflex.rb

```diff
@@ -16,11 +17,15 @@ class Circumflex < Formula
 
   depends_on "go" => :build
 
+  # Requires less 576 or later for --use-color
+  depends_on "less" if MacOS.version <= :big_sur
+
   def install
     system "go", "build", *std_go_args(output: bin/"clx", ldflags: "-s -w")
   end
 
   test do
     assert_match "List of visited IDs cleared", shell_output("#{bin}/clx clear 2>&1")
+    assert_match "Y Combinator", shell_output("#{bin}/clx view 1")
```

Comment:
```suggestion
    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"].present?

    assert_match "Y Combinator", shell_output("#{bin}/clx view 1")
```

Reply:
Alternatively, you could probably use the `PTY` module to mock a pty. Or set some environment variables to convince `less` not to panic.